manifests/b/Betterbird/Betterbird/102.6.0-bb26/Betterbird.Betterbird.installer.yaml

```diff
@@ -0,0 +1,46 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Betterbird.Betterbird
+PackageVersion: 102.6.0-bb26
+InstallerType: exe
+InstallerSwitches:
+  Silent: -ms
+  SilentWithProgress: -ms
+UpgradeBehavior: install
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerUrl: https://www.betterbird.eu/downloads/WindowsInstaller/betterbird-102.6.0-bb26.en-US.win64.installer.exe
+  InstallerSha256: 3A626BEBC9AEB4BF66C94E6F92C830E25B32DF26BD8FDE307CF89E5DBD731760
+- InstallerLocale: de-DE
+  Architecture: x64
+  InstallerUrl: https://www.betterbird.eu/downloads/WindowsInstaller/betterbird-102.6.0-bb26.de.win64.installer.exe
+  InstallerSha256: 999C68749EB8217F7A28A9F0B4D69A6590281D237FD0FB3C4EA964C3C752BAC0
+- InstallerLocale: nl-NL
+  Architecture: x64
+  InstallerUrl: https://www.betterbird.eu/downloads/WindowsInstaller/betterbird-102.6.0-bb26.nl.win64.installer.exe
+  InstallerSha256: A75B5C90176642A42561048D33D99CA3FE8308DDEBB8D17154A927A5C1280B82
+- InstallerLocale: fr-FR
+  Architecture: x64
+  InstallerUrl: https://www.betterbird.eu/downloads/WindowsInstaller/betterbird-102.6.0-bb26.fr.win64.installer.exe
+  InstallerSha256: 519836682F1FAF50D961687EE880F89B94E956292ABCA7B95C8E150712C88272
+- InstallerLocale: it-IT
+  Architecture: x64
+  InstallerUrl: https://www.betterbird.eu/downloads/WindowsInstaller/betterbird-102.6.0-bb26.it.win64.installer.exe
+  InstallerSha256: A4199D1595FDCFCCDED64513F8228CB921B8105A0A815D0116D1A3B976C648CB
+- InstallerLocale: es-AR
+  Architecture: x64
+  InstallerUrl: https://www.betterbird.eu/downloads/WindowsInstaller/betterbird-102.6.0-bb26.es-AR.win64.installer.exe
+  InstallerSha256: 13167373406C405823624C37969BCBA5C1C23005C738B367045B6A21D2AEF025
+- InstallerLocale: pt-BR
+  Architecture: x64
+  InstallerUrl: https://www.betterbird.eu/downloads/WindowsInstaller/betterbird-102.6.0-bb26.pt-BR.win64.installer.exe
+  InstallerSha256: 0B1BFA05513A590E7347705E19957268F5994AD1BA89089A579F4592AC1C4194
+- InstallerLocale: ja-JA

```

Comment:
```suggestion
- InstallerLocale: ja-ja
```

Reply:
Validation is erroring while validating the Japanese-Japan locale install. It seems to be mismatching caps, searching for `ja-ja` but finding `ja-JA` among others: 
```
2022-12-18 11:39:26.681 [CLI ] Installer [X64,exe,Unknown,ja-JA] not applicable: Installer locale does not match required locale: ja-JARequired locales: [ja-ja]
```

Additionally, there is no Betterbird.Betterbird.locale.ja-JA.yaml nor Betterbird.Betterbird.locale.ja-ja.yaml included in this manifest.